source/view.c

```diff
@@ -972,11 +1055,18 @@ static void rofi_view_refilter ( RofiViewState *state )
         tokenize_free ( state->tokens );
         state->tokens = NULL;
     }
+    if ( state->pattern ) {
```

Comment:
I must be missing something, I don't see where it is used. I just see it being set and free'ed.